GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_NASA.cfg

```diff
@@ -114,7 +114,7 @@
 	engineType = RSRMV
 }
 
-@PART[Size2LFB]:FOR[RealismOverhaul]:NEEDS[!ReStock]
+@PART[Size2LFB]:NEEDS[ReStock]:FIRST { %RestockIgnore = True }
```

Comment:
Why are we also changing this part?  The previous patch allowed ReStock to do its thing, or if you did not have ReStock then RO makes Size2LFB into something constructed

I understand changing Size3LargeTank because RO [apparently] wants to construct this model and needed to tell ReStock to stay away.  Possibly a mistake if what ReStock is making is also SLS-like and we should only rescale their work.  But I'll put that off as a problem for later.

Reply:
@DRVeyl so keep the :FOR[RealismOverhaul]:NEEDS[!ReStock]